src/resmom/linux/mom_start.c

```diff
@@ -757,7 +760,7 @@ cpusets_initialize(int recover)
  */
 
 char *
-make_cpuset(job *pjob)
+make_cpuset(job *pjob, int modify)
```

Comment:
I suggest you rename this and make it a static function. Then create two functions (or even macros) that call it (make_cpuset and modify_cpuset) where the modify parameter is set appropriately. It will make the intent of the caller easier to discern.

Reply:
Good idea! Will do.